client/merchant-feedback-prompt/negative-modal.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Loadable from 'wcpay/components/loadable';
+
+interface NegativeFeedbackModalProps {
+	onRequestClose: () => void;
+}
+
+const NegativeFeedbackModal: React.FC< NegativeFeedbackModalProps > = ( {
+	onRequestClose,
+} ) => {
+	const [ isLoading, setIsLoading ] = useState( true );
+
+	return (
+		<Modal
+			title={ __( 'Share your feedback', 'woocommerce-payments' ) }
+			className="wcpay-merchant-feedback-modal"
+			isDismissible={ true }
+			shouldCloseOnClickOutside={ false } // Should be false because of the iframe.
+			shouldCloseOnEsc={ true }
+			onRequestClose={ onRequestClose }
+		>
+			<Loadable isLoading={ isLoading }>
+				<iframe
+					title={ __(
+						'WooPayments Disable Survey',
+						'woocommerce-payments'
+					) }
+					src="https://automattic.survey.fm/woopayments-feedback-campaign-h1-2025"
```

Comment:
We'll need to keep track of this survey and manage it alongside the campaign - i.e. it's an external dependency.

Is it possible to build the modal in such a way that if the survey disappears/breaks the  modal / experience is not bad?

Reply:
I had a brief dive into iframe error handling here, but I couldn't find a reliable way to check if the iframe loaded the survey correctly:
* CORS doesn't allow us to call the URL directly and inspect the response status 200
* CORS doesn't allow us to inspect the content of the frame (e.g. check response status or "Page not found")
* The iframe will think it has loaded successfully, even if it is a "Page not found"

![image](https://github.com/user-attachments/assets/589d07f9-1de6-4d65-b577-02bb624b6d0d)

AFAICT Crowdsignal for surveys (text fields) is limited to iframes. The [API](https://crowdsignal.com/support/api/) supports voting and polls only.

Reply:
At least if we check from the Crowdsignal end we can hopefully find out if we are generating a bunch of 404 experiences. Either way we can manage this manually.

Reply:
Maybe Javascript (not iframe) can asynchronously check the iframe URL and see if the link works. Not sure if this will mess up their tracking as Crowdsignal would get hit twice.

Reply:
Gracefully degrading to just a thumb up / thumb down response sounds like a good fallback to me. If we see in tracks funnel that we are losing most people after snackbar click then we can take action. 

But maybe that's moot if we switch from Crowdsignal to tracks. 

I'm leaning towards: we can't run this campaign without using tracks, so if merchants aren't opted into tracks, we don't show the prompt.